src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -94,6 +95,40 @@ internal static IntPtr LoadNativeLibrary()
             private static void PlatformInitialize()
             {
                 LibraryResolver.EnsureRegistered();
+
+                // Check whether the version of libgdiplus is a recent version of libgdiplus.
+                try
+                {
+                    var installedVersion = LibgdiplusVersion;
+
+                    if (installedVersion == null)
+                    {
+                        throw new PlatformNotSupportedException(SR.Format(SR.LibgdiplusOutdated, LibraryRequiredVersion));
+                    }
+                    else if (installedVersion < LibraryRequiredVersion)
+                    {
+                        throw new PlatformNotSupportedException(SR.Format(SR.LibgdiplusOutdated2, LibraryRequiredVersion, installedVersion));
+                    }
+                }
+                catch (DllNotFoundException)
```

Comment:
Why catch it and not just let it propagate out? The same exception is going to be thrown immediately again as more GDI methods gets called.

Reply:
Yeah, I guess you're right. I thought there might be a corner case where `GdiplusNative.PlatformInitialize` is called but you never P/Invoke any of the libgdiplus functions but I can't come up with a realistic scenario.

At a later stage, it may be worth to catch the DllNotFoundException and provide an error message with a link to documentation on how to install libgdiplus, but we don't have that doc yet.

I removed the try/catch alltogether.